src/utilities.js

```diff
@@ -35,3 +35,20 @@ export const getBabelLoader = (config, isOutsideOfApp) => {
   }
   return babelLoader;
 };
+
+export const tap = (message, options) => (config) => {
+  const print = []
+  print.push(`Tapping the configuration`)
+  if (message) print.push(message)
+  print.push(JSON.stringify(config))
+
+  const fileDest = options && options.dest
+  if (fileDest) {
+    const fs = require('fs')
+    fs.appendFile(fileDest, `${print.join('\n')}\n`)
```

Comment:
If we're doing `appendFile`, should we add some sort of timestamp?

Also I don't have a preference one way or the other, just thinking out loud here. Does it make sense to not print the message to the console just because we're writing it to a file? Should the user get both?